bin/docker/run

```diff
@@ -5,12 +5,36 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -e
 
+# CLI handling
+
+pass_credentials_bintray=
+cmd=""
+
+while [ "$1" != "" ]; do
+    param="$1"
+
+    case $param in
+        --credentials-bintray)
+            pass_credentials_bintray=true
+            ;;
+        *)
+            cmd="$@"
+            break
+            ;;
+    esac
+
+    shift
+done
```

Comment:
I was envisioning that flags allow common credentials passing and commands like `beaver bintray` enable those flags by default.

Problem with hard coupling between such behaviour and specific commands is that it makes beaver very inflexible - either you use it exactly as supported right here and now or you can't use it at all and have to resort to manual `docker run` buildup etc.